torch/distributed/_tensor/dispatch.py

```diff
@@ -293,6 +303,12 @@ def default_tensor(spec: DTensorSpec) -> torch.Tensor:
                 cast(DTensorSpec, args_schema[0])
             ):
                 local_results = op_call(*local_tensor_args, **local_kwargs)
+        elif op_call == torch.ops.aten.convolution.default:
+            local_results = tp_convolution(op_call, *local_tensor_args, **local_kwargs)
+        elif op_call == torch.ops.aten.convolution_backward.default:
+            local_results = tp_convolution_backward(
+                op_call, *local_tensor_args, **local_kwargs
+            )
```

Comment:
@wanchaol what can we do to avoid this customized DTensor op?

Reply:
I have moved it to `convolution_backward_handler` to avoid this pollution.